ui/app/helpers/constants/routes.js

```diff
@@ -55,6 +55,64 @@ const SIGNATURE_REQUEST_PATH = '/signature-request'
 const DECRYPT_MESSAGE_REQUEST_PATH = '/decrypt-message-request'
 const ENCRYPTION_PUBLIC_KEY_REQUEST_PATH = '/encryption-public-key-request'
 
+// Used to pull a convenient name for analytics tracking events. The key must
+// be react-router ready path, and can include params such as :id for popup windows
+const PATH_NAME_MAP = {
+  [DEFAULT_ROUTE]: 'Home',
+  [UNLOCK_ROUTE]: 'Unlock Page',
+  [LOCK_ROUTE]: 'Lock Page',
+  [`${ASSET_ROUTE}/:asset`]: `Asset Page`,
+  [SETTINGS_ROUTE]: 'Settings Page',
+  [GENERAL_ROUTE]: 'General Settings Page',
+  [CONNECTIONS_ROUTE]: 'Connections Settings Page',
```

Comment:
Whoops - this route is unused. I can remove it in a separate PR.

Reply:
ill remove it from the PATH_NAME_MAP in this one.

Reply:
done

Reply:
Removed here: https://github.com/MetaMask/metamask-extension/pull/9394